### ISSUE TITLE
the pdf actual size of created pdf is too large

### ISSUE THREAD

**Issue by napasa** [manisandro/gImageReader#350]
if I have a  3400*4677 pixels image, It will output 34.00 × 46.78 inch pdf which will cause Office Word can`t show word converted for that pdf normally.

**Comment by manisandro** [comment]
Not following. The output dimensions are computed from the image pixel size and the dpi. That logic is correct, so if you want a different output size, you'll need to change one of those two: either the image pixel size or the dpi.

**Comment by napasa** [comment]
I take some time to learn some knowledge about pixel and dpi, then talk about this problem with you

**Comment by napasa** [comment]
**what is real print size:**
In printing, DPI (dots per inch) refers to the output resolution of a printer or imagesetter, and PPI (pixels per inch) refers to the input resolution of a photograph or image. DPI refers to the physical dot density of an image when it is reproduced as a real physical entity, for example printed onto paper. A digitally stored image has no inherent physical dimensions, measured in inches or centimeters. Some digital file formats record a DPI value, or more commonly a PPI (pixels per inch) value, which is to be used when printing the image. This number lets the printer or software know the intended size of the image, or in the case of scanned images, the size of the original scanned object. For example, a bitmap image may measure 1,000 × 1,000 pixels, a resolution of 1 megapixel. If it is labeled as 250 PPI, that is an instruction to the printer to print it at a size of 4 × 4 inches. Changing the PPI to 100 in an image editing program would tell the printer to print it at a size of 10 × 10 inches. However, changing the PPI value would not change the size of the image in pixels which would still be 1,000 × 1,000. An image may also be resampled to change the number of pixels and therefore the size or resolution of the image, but this is quite different from simply setting a new PPI for the file. 
from [wiki](https://en.wikipedia.org/wiki/Dots_per_inch).

**relation between Pixel resolution and Spatial resolution**
from [wiki](https://en.wikipedia.org/wiki/Image_resolution)

**Comment by napasa** [comment]
https://www.cairographics.org/ 
reduce produced pdf to normal size will make it show normally

**Comment by napasa** [comment]
you are right. and we should resize content bbox size when out paper size is not same as source
